18F/omb-eregs
# Add Python test watcher

Manual test starts
Should be rare, like a cat
Of silver and red.

`docker-compose run --rm ptw` will run the Python tests, then wait for file changes, which will kick off the tests again.
This removes the container startup time from test execution.
